code/game/objects/items/weapons/weaponry.dm

```diff
@@ -25,17 +40,161 @@
+
+	if(src && choice && !M.stat && in_range(M,src) && !M.restrained() && M.canmove && !reskinned)
+		switch(choice)
+			if(NULLROD)
+				M << "On second thought, the null rod suits you just fine."
+			if(GODHAND)
+				icon_state = "disintegrate"
+				item_state = "disintegrate"
+				name = "god hand"
+				desc = "This hand of yours glows with an aweseome power!"
+				flags = ABSTRACT | NODROP
+				w_class = 5
+				hitsound = 'sound/weapons/sear.ogg'
+				force = 20
+				damtype = BURN
+				attack_verb = list("punched", "cross counterd", "pummeled")
+			if(REDSTAFF)
+				icon_state = "godstaff-red"
+				item_state = "godstaff-red"
+				name = "holy staff"
+				desc = "It has a mysterious, protective aura."
+				w_class = 5
+				force = 5
+				slot_flags = SLOT_BACK
+				block_chance = 50
+			if(BLUESTAFF)
+				icon_state = "godstaff-blue"
+				item_state = "godstaff-blue"
+				name = "holy staff"
+				desc = "It has a mysterious, protective aura."
+				w_class = 5
+				force = 5
+				slot_flags = SLOT_BACK
+				block_chance = 50
+			if(CLAYMORE)
+				icon_state = "claymore"
+				item_state = "claymore"
+				name = "holy claymore"
+				desc = "A weapon fit for a crusade!"
+				w_class = 5
+				force = 20
+				slot_flags = SLOT_BACK|SLOT_BELT
+				block_chance = 20
+				sharpness = IS_SHARP
+				hitsound = 'sound/weapons/bladeslice.ogg'
+				attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+			if(DARKBLADE)
+				icon_state = "cultblade"
+				item_state = "cultblade"
+				name = "dark blade"
+				desc = "Spread the glory of the dark gods!"
+				slot_flags = SLOT_BELT
+				w_class = 5
+				force = 20
+				block_chance = 20
+				sharpness = IS_SHARP
+				hitsound = 'sound/weapons/bladeslice.ogg'
+				attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+			if(SORD)
+				name = "\improper SORD"
+				desc = "This thing is so unspeakably shitty you are having a hard time even holding it."
+				icon_state = "sord"
+				item_state = "sord"
+				slot_flags = SLOT_BELT
+				force = 2
+				throwforce = 1
+				w_class = 3
+				hitsound = 'sound/weapons/bladeslice.ogg'
+				attack_verb = list("attacked", "slashed", "stabbed", "sliced", "torn", "ripped", "diced", "cut")
+			if(SCYTHE)
+				icon_state = "scythe0"
+				name = "reaper scythe"
+				desc = "Ask not for whom the bell tolls..."
+				force = 15
+				w_class = 4
+				armour_penetration = 100
+				slot_flags = SLOT_BACK
+				sharpness = IS_SHARP
+				attack_verb = list("chopped", "sliced", "cut", "reaped")
+				hitsound = 'sound/weapons/bladeslice.ogg'
+				name = "mounted chainsaw"
+			if(CHAINSAW)
+				name = "chainsaw hand"
+				desc = "Good? Bad? You're the guy with the chainsaw hand."
+				icon_state = "chainsaw_on"
+				item_state = "mounted_chainsaw"
+				w_class = 5
+				flags = NODROP | ABSTRACT

```

Comment:
this will run into the issue of being deleted if the user is monkeyed since it has nothing to handle what happens when it is dropped, not sure if there's a clean fix